### DISCUSSION TITLE
Behavior of values.yaml in chart template.

### DISCUSSION THREAD

**Discussion by SrDayne** [fluxcd/flux2#3397] [General]
Hello.

Regarding docs here:
https://fluxcd.io/flux/guides/helmreleases/#refer-to-values-inside-the-chart
and here:
https://fluxcd.io/flux/components/helm/helmreleases/#specification

we can replace values by specifying file name in valuesFiles field. From docs it is not clear, should filename be exact same - "values.yaml" or we can specify any name. However,  valuesFiles works only if we specify exact file name - "values.yaml". Somebody already faced this issue:
https://github.com/fluxcd/helm-controller/issues/381

Since I don't know is it expected behavior or not, I want to left here this post for users who are looking for same answers as me.

**Comment by stefanprodan** [comment]
>  From docs it is not clear, should filename be exact same - "values.yaml" or we can specify any name.

The example that you linked has:

```yaml
      valuesFiles:
        - values.yaml
        - values-production.yaml
```

You can omit `values.yaml` and specify only `values-production.yaml` or whatever the values file is named inside your chart.

**Reply by SrDayne** [reply]
Ok, than it is a bug. 
I had main-values.yaml, dev-values-yaml and prod-values.yaml in chart. And during deploy I always had error:
`failed to locate values file: values.yaml`
And error repeated until I renamed main-values.yaml to values.yaml. Bug described here:
https://github.com/fluxcd/helm-controller/issues/381

**Reply by stefanprodan** [reply]
Ok can you please open a new issue for this in helm-controller? Thanks

**Reply by SrDayne** [reply]
Sure, issue opened: https://github.com/fluxcd/helm-controller/issues/574
